src/NonProperty.java

```diff
@@ -0,0 +1,36 @@
+/**
+ *
+ */
+public class NonProperty {
```

Comment:
This is a little complex of a class.  It should have probably been JUST the  things that weren't the property.  Then possibly another class for the ones that are the utilities and railroads